### PR TITLE
fix: [PIE-4701]:After changing the identifier in the URL, inappropriate error message is showing for Inline pipeline

### PR DESCRIPTION
**Pull request by sumanmabd**
After changing the identifier in the URL, inappropriate error message is showing for INLINE pipeline. 

#### Screenshots
 Before
Screenshot 2022-08-18 at 4 26 24 PM

After
Screenshot 2022-08-18 at 4 26 31 PM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10643)

**Comment by sumanmabd** [comment]
retrigger titlecheck

**Review by chetanvekariya-harness — DISMISSED** [review]

**Review by depanshu-dhiman — APPROVED** [review]

File: src/modules/70-pipeline/pages/pipeline-details/__tests__/PipelineDetails.test.tsx [hunk 1]
```diff
@@ -151,5 +153,105 @@ describe('Pipeline Details tests', () => {
...
+          pipelineIdentifier: 'pipeline',
+          module: 'cd'
+        }}
+        defaultAppStoreValues={defaultAppStoreValues}
+      >
+        <PipelineDetails />
+      </TestWrapper>
+    )
+
+    expect(getByText('Error occured')).toBeInTheDocument()
```
Comment (depanshu-dhiman) [inline src/modules/70-pipeline/pages/pipeline-details/__tests__/PipelineDetails.test.tsx:187]: Add 
`expect(getByText('pipeline.gitExperience.selectDiffBranch')).not.toBeInTheDocument()`

Comment (depanshu-dhiman) [reply src/modules/70-pipeline/pages/pipeline-details/__tests__/PipelineDetails.test.tsx:187]: `const updatedDefaultAppStoreValues = { ...defaultAppStoreValues, supportingGitSimplification: false }`

**Review by depanshu-dhiman — COMMENTED** [review]
Please address the comments

**Review by chetanvekariya-harness — DISMISSED** [review]

**Comment by sumanmabd** [comment]
retrigger jest

**Review by depanshu-dhiman — APPROVED** [review]

File: src/modules/70-pipeline/pages/pipeline-details/__tests__/PipelineDetails.test.tsx [hunk 2]
```diff
@@ -151,5 +153,105 @@ describe('Pipeline Details tests', () => {
...
+      refetch: jest.fn(),
+      error: {
+        data: {
+          message: 'Error occured'
+        }
+      },
+      loading: false
+    }))
+
+    jest.spyOn(commonHooks, 'useQueryParams').mockImplementation(() => ({ storeType: StoreType.REMOTE }))
```
[the remark this answers is not in the document]
Comment (sumanmabd) [reply src/modules/70-pipeline/pages/pipeline-details/__tests__/PipelineDetails.test.tsx:205]: hi @chetanvekariya-harness i tried passing query params to TestWrapper.
 
the test case is passing when we include this -->    jest.spyOn(commonHooks, 'useQueryParams').mockImplementation(() => ({ storeType: StoreType.REMOTE })) , and this is even done in other test cases which are written before.

[the remark this answers is not in the document]
Comment (chetanvekariya-harness) [reply src/modules/70-pipeline/pages/pipeline-details/__tests__/PipelineDetails.test.tsx:205]: We should continue doing what was done wrong earlier. Right way to do this is passing through TestWrapper.
Approving for now.

**Review by chetanvekariya-harness — APPROVED** [review]

**Pull request merged**